lib/vsc/utils/run.py

```diff
@@ -532,13 +521,10 @@ def _loop_process_output(self, output):
 class RunAsync(Run):
     """Async process class"""
 
-    def _prep_module(self, modulepath=None, extendfromlist=None):
+    def _prep_module(self):
         # these will provide the required Popen, PIPE and STDOUT
-        if modulepath is None:
-            modulepath = PROCESS_MODULE_ASYNCPROCESS_PATH
-        if extendfromlist is None:
-            extendfromlist = ['send_all', 'recv_some']
-        super(RunAsync, self)._prep_module(modulepath=modulepath, extendfromlist=extendfromlist)
+        from vsc.utils import assyncprocess
```

Comment:
yep,somethign is wrong with the unit tests.
They always pick up the installed vsc-base when running the tests instead of the one we want to test

this is a side effecf of changing

```
from pkgutil import extend_path

# we're not the only ones in this namespace
__path__ = extend_path(__path__, __name__)  #@ReservedAssignment
```
to the declare namespace thing. (remember 2years ago?)